packages/binary/drng/state/state.go

```diff
@@ -0,0 +1,75 @@
+package state
+
+import (
+	"encoding/binary"
+	"sync"
+	"time"
+
+	"github.com/iotaledger/hive.go/crypto/ed25519"
+)
+
+type Randomness struct {
+	Round      uint64
+	Randomness []byte
+	Timestamp  time.Time
+}
+
+// Float64 returns a float64 [0.0,1.0) rapresentation of the randomness byte slice
+func (r Randomness) Float64() float64 {
+	return float64(binary.BigEndian.Uint64(r.Randomness[:8])>>11) / (1 << 53)
+}
+
+type Committee struct {
+	InstanceID    uint32
+	Threshold     uint8
+	Identities    []ed25519.PublicKey
+	DistributedPK []byte
+}
+type State struct {
+	randomness *Randomness
+	committee  *Committee
+
+	mutex sync.RWMutex
+}
+
+func New(setters ...Option) *State {
```

Comment:
```suggestion
// New creates a new state with the given options.
func New(setters ...Option) *State {
```

I'm actually not sure whether we need the `Options`/`Option` for this, since there's only two parameters.

Reply:
Maybe we will need more options in the future

Reply:
👍